casper/src/main/resources/PoS.rhox

```diff
@@ -100,6 +105,18 @@ new  PoS,
                   }
                 }
               }
+            } |
+
+            contract PoS(@"closeBlock", ackCh) = {
+              new blockDataCh, getBlockData(`rho:block:data`), stdout(`rho:io:stdout`) in {
+                getBlockData!(*blockDataCh) |
+                for (@_, @blockNumber <- blockDataCh) {
+                  if (blockNumber % 1000 == 0) {
+                    stdout!("TODO: a validator change should happen")
+                  } |
+                  ackCh!([])
```

Comment:
What's the intended return type of `closeBlock`? If `Unit`, then `ackCh!(Nil)`

Reply:
ok. I'll use Nil